src/pages/List/ListCard.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import styled from 'styled-components';
+import 'slick-carousel/slick/slick.css';
+import Slider from 'react-slick';
+import ListImgeSlider from './ListImgeSlider';
+
+function ListCard(props) {
```

Comment:
- props 인자 자리에서 바로 구조분할당 해주세요!
```suggestion
function ListCard({ data }) {
```
- 그리고 data 라는 이름은 너무 불명확한 것 같습니다! sliderImages 라던지 어떤 데이터가 담기는지 명확하게 알 수 있도록 네이밍 해주세요!